src/components/panel/panel.spec.js

```diff
@@ -126,6 +126,35 @@ describe('$mdPanel', function() {
     expect(PANEL_WRAPPER_CLASS).not.toHaveClass(HIDDEN_CLASS);
   });
 
+
+  it('destroy should clear the config locals on the panelRef', function () {
+    openPanel(DEFAULT_CONFIG);
+
+    expect(panelRef._config.locals).not.toEqual(null);
+
+    panelRef.destroy();
+
+    expect(panelRef._config.locals).toEqual(null);
+  });
+
+  it('should focus on origin element after hide (if provided)', function () {
```

Comment:
Please add the two tests for `clickOutsideToClose` and `escapeToClose` to make sure focus is redirected there too.